assets/js/wc-gateway-ppec-smart-payment-buttons.js

```diff
@@ -268,6 +268,7 @@
 		if ( 'checkout' !== wc_ppec_context.page ) {
 			render();
 		}
+
 		$( document.body ).on( 'updated_cart_totals updated_checkout', render.bind( this, false ) );
```

Comment:
This `render.bind` is left in place because I believe this is a base js [`Function.prototype.bind()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind) and not a jQuery  [`$.bind()`](https://api.jquery.com/bind/)

Reply:
good catch :+1: